opennds/Makefile

```diff
@@ -0,0 +1,82 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=opennds
+PKG_FIXUP:=autoreconf
+PKG_VERSION:=5.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/opennds/opennds/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=opennds-$(PKG_VERSION).tar.gz
+PKG_HASH:=ee31605327b515ccecbacca8404c2ae67a47357ea0201536943031f99c96fdb6
+PKG_BUILD_DIR:=$(BUILD_DIR)/opennds-$(PKG_VERSION)
```

Comment:
This should be:
```
PKG_BUILD_DIR:=$(BUILD_DIR)/openNDS-$(PKG_VERSION)
```

Reply:
Mmm - I'm not seeing why... but if it fixes it.... 
I thought PKG_BUILD_DIR was to specify where it was to be built and could be anywhere inside BUILD_DIR within reason and not related to the package name.
I'll test it, but I am not getting the error so confusion :-D

Reply:
ok, changed it.